app/services/ManagerService.py

```diff
@@ -45,23 +46,23 @@ def TheaterOverview(self, username, filters):
 
         with connection.cursor() as cursor:
 
-            info = """select distinct MoviePlay.movName as \"Movie\", MoviePlay.movReleaseDate as \"Release_Date\", 
-            MoviePlay.movPlayDate as \"Play_Date\", Movie.duration as \"Duration\" 
-            from MoviePlay join Movie on MoviePlay.movName=Movie.movName where MoviePlay.thName in 
-            (select thName from Theater where Theater.manUsername=(%s)) 
+            info = """select distinct MoviePlay.movName as \"Movie\", MoviePlay.movReleaseDate as \"Release_Date\",
+            MoviePlay.movPlayDate as \"Play_Date\", Movie.duration as \"Duration\"
+            from MoviePlay join Movie on MoviePlay.movName=Movie.movName where MoviePlay.thName in
+            (select thName from Theater where Theater.manUsername=(%s))
             and ((%s) is NULL or Movie.movReleaseDate >=(%s))
             and ((%s) is NULL or Movie.movReleaseDate <=(%s))
             and ((%s) is NULL or MoviePlay.movPlayDate <=(%s))
-            and ((%s) is NULL or MoviePlay.movPlayDate >=(%s)) 
+            and ((%s) is NULL or MoviePlay.movPlayDate >=(%s))
             and ((%s) is NULL or Movie.duration <=(%s))
             and ((%s) is NULL or Movie.duration >=(%s))
             and ((%s) is NULL or Movie.movName like(%s))
             and ((%s) is NULL or MoviePlay.movPlayDate!=NULL)
-            Union 
-            select Movie.movName as \"Movie\", Movie.movReleaseDate as \"Release_Date\", 
+            Union
+            select Movie.movName as \"Movie\", Movie.movReleaseDate as \"Release_Date\",
             cast(NULL as date) as \"Play_Date\", Movie.duration as \"Duration\" from Movie
-            where ((%s) is NULL or Movie.movReleaseDate >=(%s)) 
-            and ((%s) is NULL or Movie.movReleaseDate <=(%s)) 
+            where ((%s) is NULL or Movie.movReleaseDate >=(%s))
+            and ((%s) is NULL or Movie.movReleaseDate <=(%s))
             and ((%s) is NULL or Movie.duration >=(%s))
             and ((%s) is NULL or Movie.duration <=(%s))
             and ((%s) is NULL or Movie.movName like(%s))"""
```

Comment:
We should probably delete these changes to TheaterOverview()

Reply:
Actually these changes look good on second look, they're just getting rid of spaces.

Reply:
These changes are just getting rid of spaces, this looks good to me.